test/Xpring/xpring-client.test.ts

```diff
@@ -160,10 +165,11 @@ describe('Xpring Client', function (): void {
 
     // WHEN a XpringClient is constructed THEN a mismatched network XpringError is thrown.
     try {
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const xpringClient = new XpringClient(payIDClient, xrpClient)
       assert.fail(
-        `Should not have been able to construct an instance of XpringClient but instead created: ${xpringClient}`,
+        `Should not have been able to construct an instance of XpringClient but instead created: ${JSON.stringify(
```

Comment:
Add a `JSON.stringify()` call because otherwise this would have just been `[object Object]` or whatever.